### DISCUSSION TITLE
785

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#957] [Comments]
# 785

좋아하는 아티스트가 2분 계시는데, 윤종신과 무라카미 하루키이다.두 분 모두 아티스트지만 번뜩이는 영감 보단 꾸준한 매일의 힘을 강조해서 굉장히 좋아한다.그러다 최근에 폴인에서 했던 윤종신님의 인터뷰를 봤다.제가 좋아하는 단어 중 하나에요.적당한 무딤?30대 초중반엔 엄청 섬세했던 것 같아요.근데 그때 무디고, 조금 덜 생각하고를 택했어요.너무 예민하고 너무 많이 알려고 하는 거를 포기했죠.그래서 이쯤에서 나 유리한 대로 그냥 생각하고 기본적으로 예상을 잘 안해요.그게 제가 약간 늦게 깨닫는 사람이라서 그런 것 같아요.조금 다른 각도의 얘기인데,제가 일찍 깨닫는 사람 별로 안 좋아해요.일찍 깨닫는 거는 그냥 그 사람이 아는 거거든요.경험하지 않고 뭔가 깨닫는 사람이 있어요.그거는 좋게 얘기하면 머리가 좋..

https://jojoldu.tistory.com/785

**Comment by Hwan0317** [comment]
예리하면 시도하지 않아 경험하는 것도 적어지고, 깨달은 채로 경험을 얻기 위해 시도하더라도 넘어졌을 때에도 그 깨달음이 반동으로 작용하기 쉽더라구요...
어려운 것을 알아도 경험한 것과 아닌 것은 분명히 다르고, 어려운 것도 '그것'이 끝인데 말이죠

주변 환경에는 조금 둔하고 무뎌지고, 눈앞의 상황에만 예리해 지는 게 좋지 않을까 합니다

오늘도 좋은 글 감사합니다.

**Comment by cremazer** [comment]
제 인생의 긴 마라톤에서 잠시 쉬어가는 시간에 동욱님의 글을 보면 많은 생각에 잠기곤 합니다. 시간이 지날수록 기술적 향상이 아니라 제 마음을 다스릴 수 있어야 한다고 생각합니다. 무던하게.. :) 천리길도 한 걸음부터..

**Comment by CityHopper** [comment]
최근에 이직을 하면서 이렇게 와닿는 글이 없었네요. 
좋은 글 감사드립니다.

**Comment by jws1837** [comment]
오늘도 어김없이 좋은 글 잘읽고 갑니다.
